images/debug/README.md

```diff
@@ -0,0 +1,47 @@
+<!--monopod:start-->
+# debug
+| | |
+| - | - |
+| **Status** | stable |
+| **OCI Reference** | `cgr.dev/chainguard/debug` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/debug/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+*[Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+Container image with the following debugging utilies included:
+- curl
+- bash
+- busybox
+- git 
+- curl
+- grpcurl
+- nmap
+- dhcping
+- tcptraceroute
+- bind-tools
+- arp-tools
+- iproute2
+- socat
+- openssl
+- gperf
+- jq
+- yq
+- strace
+- redis
+- oras
+- crane
```

Comment:
Having this list here means we need to keep it in sync with the config. Can we just say "contains some debugging utilities" and link to the config with the full list?

Reply:
totally agree with you, let me fix it!